### ISSUE TITLE
[a11y Design Review]: Ask VA

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#62429]
## Auditor(s)
- Jamie
- Kate

### Additional Support (as needed):
SK, Sara and Eli

## Tasks
`Updated as of 8/30/2023`
### Completed
- [x] Design Intent 7/28/2023
- [x] **[Design Intent Accessibility Ticket 62901 Completed](https://github.com/department-of-veterans-affairs/va.gov-team/issues/62901)** 8/3/2023
- [x] Asking to be added to design meetings for Ask Va for a11y support from CAIA 8/9/2023 - inquiry sent by @ohgreatkate
- [x] [RCVD Prototype](https://www.google.com/url?q=https://sketch.com/s/343593cd-bc08-4c6a-84d2-e3b6a5dc0c0e&sa=D&source=calendar&ust=1693088428311625&usg=AOvVaw01QZzSRzS2-bAXfgIg531Z) or wireframes needed to assist with Midpoint Review Prep ~(lo-fi) pending receipt for us to proceed with a design review prior to design intent.~
- [x] Meeting held 8/22/2023 to review current prototype, to provide CAIA feedback and Midpoint Prep. 
- [x] Ticket body 62429 will need updated to reflect more high fidelity designs, if rcvd.
- [x] Meeting Scheduled for 8/22/2023 at 12:30 p.m. EST (@ohgreatkate @sara-amanda planning to attend) - notes taken in Mural by @tygindraux 

### Pending
- [x] CAIA to provide feedback and notes, post-meeting (meeting date 8/22/2023) and after official review of the prototype, using this ticket #62429 (refer to ticket body for checklist of items to review). 
    - [x] Initial feedback provided [via the #ask-va-public Slack thread in DSVA ](https://dsva.slack.com/archives/C05A2F6DEAE/p1692656513187749)  (held 8/22/2023) by @coforma-jamie and @ohgreatkate  to @tygindraux and team.
- [ ] Schedule follow-up meeting, if needed. @tygindraux Please let us know if any additional support is needed.
- [ ] Midpoint Review (not scheduled on collab cycle calendar) as of 8/22/2023 Ref. Collaboration Cycle for Ask VA Integration - Ask VA
#62304

## CAIA Prototype Review
**Design Review (List may need updated for High-Fidelity Review Based on Prototype State)**
_Based on the development of the [prototype](https://www.google.com/url?q=https://sketch.com/s/343593cd-bc08-4c6a-84d2-e3b6a5dc0c0e&sa=D&source=calendar&ust=1693088428311625&usg=AOvVaw01QZzSRzS2-bAXfgIg531Z), some of the following may or may not be applicable._ 
- [x] [Color and contrast checks](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/prepare-for-an-accessibility-staging-review#use-of-color). Designs seem to be properly executing DSVA, no initial issues.
  - [x] WCAG 2.1 AA
  - [ ] WCAG 2.1 AAA
  - [ ] [Who Can Use](https://www.whocanuse.com/?b=ecece5&c=000000&f=20&s=)
  - [ ] High Contrast Mode
- [ ] [Keyboard navigation](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/prepare-for-an-accessibility-staging-review#keyboard-nav) not applicable – Sketch file
  - [ ] Annotations for Tab order, reflow and reading order: not yet completed – Design team reach out when ready to review these.
- [ ] Accessible Name Annotations – not applicable
  - [ ] [Alt attributes for images](https://webaim.org/techniques/alttext/)
  - [ ] Text alternative for complex images such as charts graphs maps etc.
- [x] Language (ease of reading) – A11y team recommends a content review when copy is finalized
- [ ] HTML Tags – not applicable – can review when annotations are ready
  - [ ] [Page Title](https://accessibility.18f.gov/page-titles/)
  - [ ] [Heading order](https://accessibility.18f.gov/headings/)
   - [ ] Annotations (if needed)
  - [ ] Semantic Consideration for paragraphs ordered lists numbering
  - [ ] [Tables (heading rows)](https://design.va.gov/components/table)
- [ ] Size of clickable areas – not applicable yet

### As Time Allows
- [ ] [Components](https://design.va.gov/components/) follow VA design system guidance.
- [ ] [Patterns](https://design.va.gov/patterns/) follow VA design system guidance.

## Prototype Link

[Link to prototype](https://www.google.com/url?q=https://sketch.com/s/343593cd-bc08-4c6a-84d2-e3b6a5dc0c0e&sa=D&source=calendar&ust=1693088428311625&usg=AOvVaw01QZzSRzS2-bAXfgIg531Z) `Version 8/21/2023`

## CAIA Design Review Feedback
**_Based on the prototype review above in the areas mentioned above._** 

#### Concise issue description
- Where you found it
- More details

##### Bandaid solution
- If helpful, provide a temporary solution that involves less content/design changes

##### Recommended solution
- Try to provide an ideal solution too that explores alternatives should content, design, IA, the broader service, etc can be fixed
- If no research has been conducted with assistive technology users, it should be a priority

#### Concise issue description
- Where you found it
- More details

##### Bandaid solution
- If helpful, provide a temporary solution that involves less content/design changes

##### Recommended solution
- Try to provide an ideal solution too that explores alternatives should content, design, IA, the broader service, etc can be fixed
- When possible, point to more research with assistive technology users

#### Concise issue description
- Where you found it
- More details

##### Bandaid solution
- If helpful, provide a temporary solution that involves less content/design changes

##### Recommended solution
- Try to provide an ideal solution too that explores alternatives should content, design, IA, the broader service, etc can be fixed
- When possible, point to more research with assistive technology users

**Comment by sara-amanda** [comment]
## CAIA Next Steps
@eli-oat @coforma-jamie @coforma-terry 
- [x] #60696 converted epic
- [x] #60696 tagged as an epic
- [x] Design review ticket created (this ticket) and tied to the epic (#60696)
- [x] Estimated work at 2 hours (please change if you feel it should be different (options: 2, 4, 8)
- [x] Related links added in notes at the top of this ticket
- [x] Design Intent - scheduled for 7/28/2023
- [ ] **Prototype or wireframes (lo-fi) pending receipt for us to proceed with a design review prior to design intent.**

**Comment by sara-amanda** [comment]
@coforma-terry moving this to "New Intake Requests" from no status in Projects to match ZenHub "New Issues."

**Comment by sara-amanda** [comment]
## CAIA a11y Next Steps Update
@eli-oat @coforma-terry @coforma-jamie @ohgreatkate @SarahKay8 

### Completed
- [x] Design Intent 7/28/2023
- [x] **[Design Intent Accessibility Ticket 62901 Completed](https://github.com/department-of-veterans-affairs/va.gov-team/issues/62901)** 8/3/2023
### Pending
- [ ]  Prototype or wireframes needed to assist with Midpoint Review Prep (~) (lo-fi) pending receipt for us to proceed with a design review prior to design intent. (~)
- [ ] Ticket body 62429 will need updated to reflect more high fidelity designs, if rcvd.
- [ ] Asking to be added to design meetings for Ask Va for a11y support from CAIA
- [ ] Midpoint Review (not scheduled on collab cycle calendar as of 8/9/2023

**Comment by sara-amanda** [comment]
## CAIA a11y Next Steps Update
@eli-oat @coforma-terry @coforma-jamie @ohgreatkate @SarahKay8 

### Completed
- [x] Design Intent 7/28/2023
- [x] **[Design Intent Accessibility Ticket 62901 Completed](https://github.com/department-of-veterans-affairs/va.gov-team/issues/62901)** 8/3/2023
- [x] Asking to be added to design meetings for Ask Va for a11y support from CAIA 8/9/2023 - inquiry sent by @ohgreatkate
- [x] [RCVD Prototype](https://www.google.com/url?q=https://sketch.com/s/343593cd-bc08-4c6a-84d2-e3b6a5dc0c0e&sa=D&source=calendar&ust=1693088428311625&usg=AOvVaw01QZzSRzS2-bAXfgIg531Z) or wireframes needed to assist with Midpoint Review Prep ~(lo-fi) pending receipt for us to proceed with a design review prior to design intent.~
- [x] Meeting held 8/22/2023 to review current prototype, to provide CAIA feedback and Midpoint Prep. 
- [x] Ticket body 62429 will need updated to reflect more high fidelity designs, if rcvd.
- [x] Meeting Scheduled for 8/22/2023 at 12:30 p.m. EST (@ohgreatkate @sara-amanda planning to attend) - notes taken in Mural by @tygindraux 

### Pending
- [ ] CAIA to provide feedback and notes, post-meeting  (held 8/22/2023) to @tygindraux via this ticket
- [ ] Schedule follow-up meeting, if needed.
- [ ] Midpoint Review (not scheduled on collab cycle calendar) as of 8/22/2023 Ref. Collaboration Cycle for Ask VA Integration - Ask VA
#62304

**Comment by ohgreatkate** [comment]
@coforma-jamie and I attended a design review call with the VFS product team. We provided feedback and additional resources in a follow-up thread in [Slack](https://dsva.slack.com/archives/C05A2F6DEAE/p1692656513187749). 

@tygindraux Please let us know if prototypes are being made later or if your team would like any additional reviews, support on HTML tags, and/or accessible annotations.

**Comment by ohgreatkate** [comment]
Additional notes:

- Progress bar with steps language, varies from known DSVA patterns but seems beneficial to our Veterans. Nice.
   - Not including "Step X of Y" may lose Veterans' confidence to where they are in the form, recommend testing to validate if this adjusted pattern is successful
   - Be mindful of the Step in circle visual pattern and how that would present for Veterans using AT
- Review Headings and language for consistency and to follow design style guide. eg. sentence case, language consistency, punctuation, and plain language (inquiry?)

**Comment by tygindraux** [comment]
Thanks for this feedback Kate! Super helpful, I'll bring it back to the team.

Thomas is working on finishing up a prototype in Sketch right now, prepping for midpoint review. I'm sure we'll come back to you for another review and/or help with annotations and tags. Thank you!
